src/moduleBuild.js

```diff
@@ -90,6 +90,14 @@ function buildUMD(args, buildConfig, userConfig, cb) {
       spinner.fail()
       return cb(err)
     }
+
+    if (userConfig.uglify === false) {
+      spinner.succeed()
```

Comment:
This is duplicated from below. I haven't tested the implications of sharing that earlier on (probably above this line) on the output, but happy to try if you think it's better to push this above.